google-cloud-webrisk/README.md

```diff
@@ -1,5 +1,10 @@
 # Ruby Client for Web Risk API
 
+**This library is deprecated, and will no longer receive updates. Please use the
+[`google-cloud-web_risk`](https://rubygems.org/gems/google-cloud-web_risk)
+library instead. For detailed information on the differences, see the
+[migration guide](https://googleapis.dev/ruby/google-cloud-web_risk/latest/file.MIGRATING.html).**
```

Comment:
This link returns 404. (I'm sure releasing `google-cloud-web_risk` is part of the plan; just noting the current state.)

Reply:
link is now active.